specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/2024-07-26-preview/examples/ledger/PatchUserDefinedRole.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parameters": {
+    "api-version": "2024-07-26-preview",
+    "role": {
+      "roleName": "administrator",
+      "roleAction": [
+        "Microsoft.ConfidentialLedger/ledger/users/write"
```

Comment:
What is the behavior of patch in this case? Is it going to replace or append the existing actions?

Reply:
Patch here will replace the allowed actions on an existing role